models/cable/inst/template.job

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# redirect output
+exec 3>&1
+exec &> "@OUTDIR@/logfile.txt"
+
+# host specific setup
+@HOST_SETUP@
+
+# create output folder
+mkdir -p "@OUTDIR@"
+
+# see if application needs running
+if [ ! -e "@OUTDIR@/results.csv" ]; then
```

Comment:
Unless a previous run of CABLE produces a results.csv file (seems unlikely) this test won't ever cause previous runs to be skipped. Probably best for now to comment out this line and the matching ```fi``` on line 30